controllers/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -99,6 +105,181 @@ def test_close(self):
         self.svc.disconnect()
         self.svc.client.close.assert_called_with()
 
+    def test_get_replication_success(self):
+        replication_request = ReplicationRequest(OBJECT_INTERNAL_ID, OTHER_OBJECT_INTERNAL_ID, SYSTEM_ID, COPY_TYPE,
+                                                 REPLICATION_TYPE_MIRROR)
+        rcrelationship_to_return = \
+            Munch({svc_settings.RCRELATIONSHIP_STATE_ATTR_NAME: RCRELATIONSHIP_STATE_READY,
+                   svc_settings.RCRELATIONSHIP_COPY_TYPE_ATTR_NAME: svc_settings.RCRELATIONSHIP_COPY_TYPE,
+                   svc_settings.RCRELATIONSHIP_MASTER_CLUSTER_ID_ATTR_NAME: "",
+                   svc_settings.RCRELATIONSHIP_PRIMARY_ATTR_NAME: False,
+                   svc_settings.RCRELATIONSHIP_NAME_ATTR_NAME: ""})
+        self.svc._get_rcrelationship = Mock()
+        self.svc._get_rcrelationship.return_value = rcrelationship_to_return
+
+        self.svc.get_replication(replication_request)
```

Comment:
please assert that you get what you want and that the ls called with the right args

Reply:
the fact that a method didn't raise any error didn't mean that it did anything

Reply:
Done